react/storybook-v6/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "storybook-v6",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@invisionapp/dsm-storybook": "latest",
+    "@storybook/addon-essentials": "^6.0.0",
+    "@storybook/addon-knobs": "^6.0.21",
+    "@storybook/addon-links": "^6.0.0",
+    "@storybook/addon-options": "^5.3.0",
+    "@storybook/addons": "^6.0.0",
+    "@storybook/node-logger": "^6.0.0",
+    "@storybook/preset-create-react-app": "^3.1.4",
+    "@storybook/react": "^6.0.0",
+    "acorn": "^6.1.1",
+    "core-js": "^2.6.5",
+    "prop-types": "15.6.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-scripts": "3.0.0",
+    "react-svg": "^11.0.40"
```

Comment:
One thing I want to note is that I switched to `react-svg` from `react-inline-svg`. For some reason in newer versions of Storybook webpack is handling module-level images/svgs a little differently, and `react-svg-inline` wasn't handling it correctly and was instead just rendering the internal Webpack static asset path instead of the actual SVG contents.

`react-inline-svg` wasn't working out of the box and I didn't want to spend a lot of time investigating webpack issues. Now we use `react-svg`.